win32ss/user/ntuser/simplecall.c

```diff
@@ -505,9 +506,25 @@ NtUserCallTwoParam(
         }
 
         case TWOPARAM_ROUTINE_SWITCHTOTHISWINDOW:
-            STUB
+        {
+            HWND hwnd = (HWND)Param1;
             Ret = 0;
+            Window = UserGetWindowObject(hwnd);
+            if (!Window)
+            {
+                break;
+            }
+            hwnd = Window->head.h;
+            if ((BOOL)Param2)
+            {
+                if (Window->style & WS_MINIMIZE)
+                {
+                    NtUserPostMessage(hwnd, WM_SYSCOMMAND, SC_RESTORE, 0);
```

Comment:
`NtuserPostMessage` calls `UserEnterExclusive`, but that is also called already at the start of this function.
So you should probably use `UserPostMessage` instead.